cmd/user-auth/main_test.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/edulinq/autograder/internal/cmd"
+	"github.com/edulinq/autograder/internal/db"
+	"github.com/edulinq/autograder/internal/exit"
+)
+
+const SUCCESS_AUTH = `{
+    "success": true
+}
+`
```

Comment:
Since this is expected test data and not a true constant, lower case it and put it at the bottom of the file.

Also change the name to indicate that it is expected output.